doc/user-docs/customization.md

```diff
@@ -0,0 +1,65 @@
+# Windows Terminal Customization
+
+NOTE: At the time of writing Windows Terminal is still under active development and many things will
+change. If you notice an error in the docs, please raise an issue. Or better yet, please file a PR with an appropriate update!
+
+## Title of tab
+When Windows Terminal launches, the shell has a special variable to determine the window title. This title is used to set the name of the tab inside Windows Terminal.
```

Comment:
Suggest something like: 

"When a tab is created in Windows Terminal, its title is set either by the app/shell to which the Terminal's tab is connected, or via the related profile's settings in the Terminal's `profiles.json` file.